pepper-apis/src/appengine/DataDonationPlatform.yaml

```diff
@@ -2,17 +2,16 @@ service: pepper-backend
 runtime: java11
 instance_class: F4
 
-# @todo back to 2 when dev completed
 automatic_scaling:
   max_instances: 4 #up to 2147483647
   min_instances: 0 #0 means first one starts only on first request. Needs additional setup to use
   max_idle_instances: 1 #default is automatic
   min_idle_instances: 0 #also needs additional setup
   target_cpu_utilization: 0.8 #0.6 is the default. Higher might trigger new instances created
   target_throughput_utilization: 0.8 #default is 0.6 . Also metric to launch more instances
-  max_concurrent_requests: 10 #default is 10 . Max is 80. GCP recommends 10 max value.
-  max_pending_latency: 60ms #Default is 30ms. Max time a request is allowed to wait in queue.
-  min_pending_latency: 60ms #Default is 30ms. Threshold used to scale down
+  max_concurrent_requests: 30 #default is 10 . Max is 80
+  max_pending_latency: 15000ms #Default is 30ms. Max time a request is allowed to wait in queue.
+  min_pending_latency: 12000ms #Default is 30ms. Threshold used to scale down
 network:
```

Comment:
Some of our requests take a second or more to complete, so the ms scale settings made no sense for Pepper. We can tolerate more concurrent calls. After reading docs the automatic scaling does seem so crazy. Being that it is the cheapest to operate, probably makes sense to kick tires a bit before switching to something else.